src/sentry/search/eap/ourlogs/attributes.py

```diff
@@ -51,6 +51,13 @@
         ),
         simple_sentry_field("browser.name"),
         simple_sentry_field("environment"),
+        simple_sentry_field("message.template"),
+        simple_sentry_field("release"),
+        simple_sentry_field("trace.parent_span_id"),
+        simple_sentry_field("sdk.name"),
+        simple_sentry_field("sdk.version"),
+        simple_sentry_field("origin"),
+        simple_sentry_field("server.address"),
```

Comment:
```suggestion
```

This is always sent as `server.address`, not `sentry.server.address`, so we don't need to alias it.